### ISSUE TITLE
Autocomplete in interactive window not working for user module

### ISSUE THREAD

**Issue by darim1** [microsoft/vscode-jupyter#4652]
I am encountering problems, when trying to get autocomplete working for my custom module.

This is the module:

```
print("Hello world")
a = 1
b = 1
c = 3
def Test():
    print('Test')
import scipy
scipy.fft
```
Autocomplete works fine, on the editor side. It also works fine in the interactive window, when completing builtin modules like scipy. But somehow I cannot manage it the following way:

```
import mymodule
 
mymodule.  # tab or cntrl-space
```
Autocomplete would only suggest mymodule.py, which is useless in this case.

I've tried several different engines, like kite or intellicode. None of them seems to be able to resolve the module's content, when queried from the interactive window.

I am using vscode 1.52.1 with the respective extensions.

Is this a bug or a settings problem?

Thanks,
Daniel

**Comment by rchiodo** [comment]
Are you using pylance? This should work if pylance is installed.

This is what I get.:
![image](https://user-images.githubusercontent.com/19672699/107428448-8235aa00-6ad7-11eb-94e4-8519aca0a0fc.png)

**Comment by darim1** [comment]
Nope. Pylance does not do the trick. I also tried intellicode and kite without success.

It works, however, for built in modules e.g. numpy, but not for my own module.

Am 9. Februar 2021 22:05:58 MEZ schrieb Rich Chiodo <notifications@github.com>:
>Are you using pylance? This should work if pylance is installed.
>
>This is what I get.:
>![image](https://user-images.githubusercontent.com/19672699/107428448-8235aa00-6ad7-11eb-94e4-8519aca0a0fc.png)
>
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/microsoft/vscode-jupyter/issues/4652#issuecomment-776243804
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by rchiodo** [comment]
Where is mymodule in relation to the other files in the workspace? 

Additionally you have to wait for pylance to finish loading. For me this took like 2 minutes. I verified it was working with a regular python file first.

**Comment by darim1** [comment]
The file is in my project folder. Waiting doesn't enhance the situation. Please have a look at the sceenshot.

![grafik](https://user-images.githubusercontent.com/20026777/107487622-a7cdbc80-6b86-11eb-9c05-31d597795eb6.png)

Thx

**Comment by rchiodo** [comment]
Can you list out your VS code, python, jupyter, and pylance extension versions?

And verify that pylance is the selected language server?

Should be here:

![image](https://user-images.githubusercontent.com/19672699/107545547-46a2ea80-6b80-11eb-81e2-bf49f230f1bf.png)

**Comment by darim1** [comment]
Version: 1.53.1 (user setup)
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.18362
_______________________
Started 'Python 3.6.12 64-bit ('py36': conda)' kernel
Python 3.6.12 |Anaconda, Inc.| (default, Sep  9 2020, 00:29:25) [MSC v.1916 64 bit (AMD64)]
______________________
Jupyter: v2020.12.414227025
______________________
pylance: v2021.2.0
______________________
The language server is set to pylance.
______________________
I've also tried the other servers, but without success.
